examples/01_dataset_simulation.ipynb

```diff
@@ -46,7 +46,7 @@
     {
```

Comment:
<p>is tex only failing here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/radionets-project/radionets/pull/109/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
<p>know it is working for me aswell</p>